observability/src/main/java/io/grpc/observability/interceptors/LogHelper.java

```diff
@@ -300,53 +317,85 @@ void logCancel(
         .setEventLogger(eventLogger)
         .setLogLevel(LogLevel.LOG_LEVEL_DEBUG)
         .setRpcId(rpcId);
-    sink.write(logEntryBuilder.build());
+    sink.write(logEntryBuilder.build(), locationTags, customTags);
```

Comment:
Pls see my other comment about setting the locationTags, customTags just once in the sink.